### ISSUE TITLE
get all teamMembers before rendering meeting components

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/action#136]
We store the `checkInOrder` on teamMembers, and each one can come it at any time.
It's important to wait until they all come in before we render because
- we want folks to see our ducks (only kinda kidding)
- our checkin-card logic requires `localPhaseItem` as well as +/- 1.
- we shouldn't allow someone to go to `/checkin/100`, rather we should redirect them to `/checkin/:facilitatorPhaseItem`.

RethinkDB just added a new feature called `includeType` which gives us everything we need (includeState, includeInitial) and so much more (the bespoke add/update/remove logic I wrote).
https://rethinkdb.com/docs/changefeeds/javascript/
I think this will solve all of our problems.

**Comment by mattkrick** [comment]
TL;DR, kept logic at the application layer.

To handle the `status` of the subscription requires more thought.
Maybe that should be kept in redux store? if so, where?
For now, cashay has a `setStatus` handler that someone can use in their subscriber. I think this'll be the best approach, but it's working for now!
